apps/bfd-model/bfd-model-codegen/src/main/java/gov/cms/bfd/model/codegen/RifLayoutsProcessor.java

```diff
@@ -1503,4 +1828,195 @@ private void writeDebugLogMessages() {
       throw new UncheckedIOException(e);
     }
   }
+
+  /**
+   * Creates the fields for the BeneficiaryMonthly class in the model rif
+   *
+   * @param lineEntity helps build the entity {@link TypeSpec.Builder}
+   * @param isId determines if the field is an id field
+   * @param isTransient determines if the field is transient {@link boolean}
+   * @param isColumnOptional determines if the field is optional {@link boolean}
+   * @param fieldName specifies the fieldname {@link String}
+   * @param type specifies the field type {@link RifColumnType}
+   * @param columnLength specifies the column length {@link Optional<Integer>}
+   * @param columnScale specifies the column scale {@link Optional<Integer>}
```

Comment:
Would you mind applying this suggestion?
```suggestion
   * @param columnLength specifies the column length {@link Optional<Integer>}, for numeric types this represents the total number of digits that can be stored
   * @param columnScale specifies the column scale {@link Optional<Integer>}, for numeric types this represents how many of the total digits (see `columnLength`) are to the right of the decimal point
```